### ISSUE TITLE
nRF52-DK broken on `master`

### ISSUE THREAD

**Issue by niklasad1** [tock/tock#1079]
Hey,

`nRF52-DK` seems to be broken on the latest master (f695769e34b2b67073c7f02518b166c537111359)

I can see the following print: 
```bash
Initialization complete. Entering main loop
NRF52-DK HW INFO: Variant: AAB0, Part: N52832, Package: QF, Ram: K64, Flash: K512
```

But then `LED3` and `LED4` lights up even without any `app` loaded!

Anyone that can produce this?

**Comment by alevy** [comment]
I was able to reproduce this. I haven't no idea what's going on, though if I had to guess, it's to do with the common nrf52-base board making an incorrect assumption about PIN assignments (or something like that---e.g. pulling those pins low because it's thinks they are buttons or something like that).

It doesn't impact other functionality, though. I'm able to run applications just fine and the work other than those two LEDs not being functional and just staying on.
